### ISSUE TITLE
Editing ListViews

### ISSUE THREAD

**Issue by calvinwyoung** [simple-salesforce/simple-salesforce#209]
Hi all,

Apologies in advance if this is out of the scope of this project, but is this API client able to update ListViews programmatically?

**Comment by andscoop** [comment]
@calvinwyoung Please review #123 and re-open if following those steps does not resolve your need.

**Comment by skamensky** [comment]
@andscoop I think @calvinwyoung was trying to update list views, not view the metadata - which is what issue #123 addresses. As far as I know, updating metadata can't be done with the REST API. 

According to Salesforce [documentation](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/using_resources_working_with_object_metadata.htm)

> [U]se REST API resources to retrieve object metadata information. For modifying or creating object metadata information, see the Metadata API Developer Guide.

**Comment by calvinwyoung** [comment]
@andscoop @skamensky Yes, I was hoping to edit the ListView. Does this SDK only interact with the REST API? are there any plans to interact with the Metadata API?
